docs/api.md

```diff
@@ -169,7 +169,7 @@ module LikeDog = [%graphql
 [@react.component]
 let make = () => {
   let ({ response }, _) = useQuery(~request=GetAllDogs.make(), ());
-  let (_, executeMutation) = useMutation(~query=LikeDog.query, ~parse=LikeDog.parse);
```

Comment:
This was just left over from, I think, a past API proposal. But it's quite incorrect, so I made sure to correct it while here 😅